clients/js/cmds/sui.ts

```diff
@@ -16,6 +17,31 @@ function assertNetwork(n: string): asserts n is Network {
   }
 }
 
+/*
+  Loop through a list of Sui objects and look for the DeployerCapability that should
+  have been granted upon publication of the package with package_id
+
+  The objects is in the format returned by a json-rpc call "provider.getObjectsOwnedByAddress"
+*/
+function findDeployerCapability(package_id: string, module_name: string, objects: Array<any>): string {
+      let deployer;
+      for (var obj of objects){
+        let t = obj.type;
+        // do a basic check and see if the type is as expected (has a normal package, module, type name separated by "::")
+        var count = [...t].filter(x => x === ":").length;
+        if (count==4){
+          let t_split = t.split("::") // load the package, module, type name in an array
+          if (t_split[0].toUpperCase() == package_id.toUpperCase()){
```

Comment:
can we construct the fully qualified type given by `packageId::moduleName::DeployerCapability` and just string match? logic would be much cleaner than having to split the string and conditionally matching on tokens

Reply:
maybe something like
```
function findDeployerCapability(packageId: string, moduleName: string, objects: any[]): string | null {
  const type = `${packageId}::${moduleName}::DeployerCapability`;
  return objects.find(o => o.type.toLowerCase() === type.toLowerCase())?.objectId ?? null;
}
```

Reply:
very clean